example/src/pages/camera/camera.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, NgZone } from '@angular/core';
 import { DomSanitizer, SafeResourceUrl } from '@angular/platform-browser';
-import { IonicPage, NavController, NavParams } from 'ionic-angular';
+import { IonicPage, NavController, NavParams, AlertController } from 'ionic-angular';
```

Comment:
```suggestion
import { IonicPage, NavController, NavParams } from 'ionic-angular';
```
remove unused `AlertController`